modules/network-observability-flowcollector-api-specifications.adoc

```diff
@@ -217,6 +217,18 @@ This feature requires mounting the kernel debug filesystem, so the eBPF agent po
 It requires using the OVN-Kubernetes network plugin with the Observability feature.
 IMPORTANT: This feature is available as a Developer Preview. +
 
+- `PacketTranslation`: enable enriching flows with packet's translation information.  +
```

Comment:
```suggestion
- `PacketTranslation`: enable flow enrichment with packet's translated information.  +
```

Reply:
Yes and additionally line 218 is for Network Events, which is now in Technology Preview, so that needs to be updated. I commented in Joel's regeneration PR to update it.